cmd/crio/main.go

```diff
@@ -303,6 +306,11 @@ func main() {
 			Value: "error",
 			Usage: "log messages above specified level: debug, info, warn, error (default), fatal or panic",
 		},
+		cli.StringFlag{
+			Name:  "log-dir",
+			Value: "",
+			Usage: "default log directory were all logs will go unless directly specified by the kubelet",
```

Comment:
nit: same as above, where instead of were

Reply:
Fixed :)